src/main/java/org/minions/devfund/katerinaanzoleaga/battleship/Ocean.java

```diff
@@ -0,0 +1,283 @@
+    private static final int DESTROYERSNUMBER = 3;
+    private static final int SUBMARINESNUMBER = 4;
+
+    private Ship[][] ships = new Ship[LENGTH][LENGTH];
+
+    /**
+     * Returns the Ships array.
+     * @return Array of ships
+     */
+    public Ship[][] getShipArray() {
+        return ships.clone();
+    }
+
+    /**
+     * Updates the Ships array.
+     * @param ships of the ocean.
+     */
+    public void setShips(final Ship[][] ships) {
+        this.ships = ships.clone();
+    }
+
+    /**
+     * Returns the number of shots the gamer fired.
+     * @return shots fired.
+     */
+    public int getShotsFired() {
+        return shotsFired;
+    }
+
+    /**
+     * Returns the number of hits.
+     * @return hit count
+     */
+    public int getHitCount() {
+        return hitCount;
+    }
+
+    /**
+     * Returns the number of Ships already sunk.
+     * @return shis sunk
+     */
+    public int getShipsSunk() {
+        return shipsSunk;
+    }
+
+    /**
+     * Ocean constructor.
+     */
+    public Ocean() {
+        shotsFired = 0;
+        hitCount = 0;
+        shipsSunk = 0;
+        this.fillOcean();
+    }
+
+    /**
+     * Fills the Ocean with empty ships.
+     */
+    private void fillOcean() {
+        for (int i = 0; i < this.ships.length; ++i) {
+            for (int j = 0; j < this.ships.length; ++j) {
+                EmptySea emptySea = new EmptySea();
+                emptySea.setBowRow(i);
+                emptySea.setBowColumn(j);
+                this.ships[i][j] = emptySea;
+            }
+        }
+    }
+
+    /**
+     * Prints the ocean with the values to display the game board and ships' status.
+     *
+     */
+    public void print() {
+        String lineSeparator = System.lineSeparator();
+        StringBuilder sb = new StringBuilder();
+        Formatter fmt = new Formatter(sb);
+        int hitPosition;
+
+        final String spaces = " ";
+        for (int i = -1; i < this.ships.length; ++i) {
+            for (int j = -1; j < this.ships.length; ++j) {
+                if (i == -1 && j == -1) {
+                    fmt.format("%2s", "");
+                }
+                if (i == -1 && j != -1) {
+                    sb.append(spaces);
+                    fmt.format("%02d", j);
+                }
+                if (j == -1 && i != -1) {
+                    fmt.format("%02d", i);
+                }
+                if (i != -1 && j != -1) {
+                    hitPosition = this.ships[i][j].isInShipPosition(i, j);
+                    if (hitPosition != -1) {
+                        if (this.ships[i][j].getHit()[hitPosition]) {
+                            fmt.format("%3s", this.ships[i][j].toString());
+                        } else {
+                            fmt.format("%3s", ".");
+                        }
+                    }
+                }
+            }
+            sb.append(lineSeparator);
+        }
+        System.out.println(sb.toString());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)